digitalpy/core/zmanager/action_mapper.py

```diff
@@ -0,0 +1,10 @@
+from abc import ABC
+from digitalpy.core.zmanager.request import Request
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>I003 isort expected 1 blank line in imports, found 0